lib/index.js

```diff
@@ -74,17 +74,10 @@ export default {
 
         // Call ERB to "de-templatize" the code
         return helpers.exec(this.erbPath, erbArgs,
-          { stdin: text, cwd: fileDir }
+          { stdin: text.replace(/<%=/g, '<%'), cwd: fileDir }
```

Comment:
I believe the other scope comes from a different language package which you may not have installed? Not sure though.

If the replacement causes no issues with regular files, and doesn't prevent any errors from being caught then it shouldn't be an issue leaving it in.